packages/IntegrationTest/windows/integrationtest/TestCommandListener.cpp

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "TestCommandListener.h"
+
+#include <winrt/Windows.Storage.Streams.h>
+
+using namespace winrt::integrationtest;
+using namespace winrt::Microsoft::ReactNative;
+using namespace winrt::Windows::Data::Json;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Networking;
+using namespace winrt::Windows::Networking::Sockets;
+using namespace winrt::Windows::Storage::Streams;
+
+namespace IntegrationTest {
+
+IAsyncAction TestCommandResponse::Okay() noexcept {
+  JsonObject responseJson;
+  responseJson.SetNamedValue(L"status", JsonValue::CreateStringValue(L"okay"));
+  co_await SendJson(responseJson);
+}
+
+IAsyncAction TestCommandResponse::Timeout() noexcept {
+  JsonObject responseJson;
+  responseJson.SetNamedValue(L"status", JsonValue::CreateStringValue(L"timeout"));
+  co_await SendJson(responseJson);
+}
+
+IAsyncAction TestCommandResponse::TestPassed(bool passed) noexcept {
```

Comment:
I should change this to `TestCompleted` or something along those lines and give a different status from Okay.